test/cpp/tensorexpr/test_kernel.cpp

```diff
@@ -210,7 +210,11 @@ TEST_F(Kernel, Huge) {
   TensorExprKernel k(graph);
   std::ostringstream oss;
   oss << *k.getCodeGenStmt();
-  const std::string& verification_pattern = "# CHECK: 4000000000";
+  // The 4000000000 iterations loop will be split into 500000000 x 8 and the
+  // outer loop will be parallel. If LLVM is not present, it will not be split,
+  // and to cover both of these cases we're looking for 00000000ll; in the
+  // output.
+  const std::string& verification_pattern = R"IR(# CHECK: 00000000ll;)IR";
```

Comment:
nit: we don't quite have to check for that number. We just want to make sure that the compilation is successful here. So, even checking for something like `aten_relu` would suffice..

Reply:
I wanted to still have a check for a number to make sure we're not truncating it or something. But if it happens to be too fragile, we can indeed just check that it compiles.